tests/src/test/RegisterAddress.spec.ts

```diff
@@ -0,0 +1,55 @@
+// [object Object]
+// SPDX-License-Identifier: Apache-2.0
+
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api';
+
+describe('RegisterAddress', (): void => {
+  let api: ApiPromise;
+  let alice;
```

Comment:
@nathanwhit, @jfrank-summit I found some examples but didn't find a searchable API reference on the polkadot website. I was looking for a mention of the data type of this variable. :-(. 

In case I've missed that can you post a link.

Reply:
Alice should be something like:
```
const keyring = new Keyring({ type: 'sr25519' });
const alice = keyring.addFromUri('//Alice', { name: 'Alice' });
```